src/firebaseConf.ts

```diff
@@ -14,7 +14,12 @@ import { getStorage } from "firebase/storage";
 import { Zoom, toast } from "react-toastify";
 
 const firebaseConfig = {
-  // firebase configuration
+  apiKey: "AIzaSyD2v9AIAjzNaQuS5-GJufDquaTO-9gPMUc",
```

Comment:
@Dhevamuthu Remove this. Never push your firebase keys, they are confidential.

Reply:
Will do it